webapp-django/crashstats/authentication/management/commands/auditgroups.py

```diff
@@ -46,26 +51,30 @@ def audit_hackers_group(self, dryrun):
             elif not user.email.endswith(VALID_EMAIL_DOMAINS):
                 users_to_remove.append((user, 'invalid email domain'))
             elif user.last_login and user.last_login < cutoff:
+                days = delta_days(user.last_login)
+
                 # This user is inactive. Check for active API tokens.
                 active_tokens = [
                     token for token in user.token_set.all()
                     if not token.is_expired
                 ]
                 if not active_tokens:
-                    users_to_remove.append((user, 'inactive since cutoff, no tokens'))
+                    users_to_remove.append((user, 'inactive %sd, no tokens' % days))
                 else:
                     self.stdout.write(
-                        'SKIP: {} (inactive, but has active tokens: {})'.format(
-                            user.email, len(active_tokens)
+                        'SKIP: %s (inactive %sd, but has active tokens: %s)' % (
+                            user.email, days, len(active_tokens)
                         )
                     )
 
         # Log or remove the users that have been marked
-        for user, reason in users_to_remove:
-            self.stdout.write('Removing: {} ({})'.format(user.email, reason))
+        for user, reason in sorted(users_to_remove, key=lambda item: (item[1], item[0])):
```

Comment:
I just realized this isn't going to work because the number of days is variable. I'll nix it.